test/compiler-finder-tests.js

```diff
@@ -119,7 +119,7 @@ describe('Compiler-finder', function () {
 
     it('should not hang for undefined groups (Bug #860)', () => {
         const finder = new CompilerFinder({}, compilerProps, properties.fakeProps({}), {}, optionsHandler);
-        return finder.getCompilers().should.eventually.have.lengthOf(2);
+        return finder.getCompilers().should.eventually.have.lengthOf(1);
```

Comment:
Before the changes I've made to CompilerFinder `finder.getCompilers()` here was returning a `[{ ... actual compiler ... }, []]`. Now it just returns `[{ ... actual compiler ... }]`.